include/exadg/utilities/throughput_parameters.h

```diff
@@ -27,10 +27,8 @@
 #  include <likwid.h>
 #endif
 
-// deal.II
-#include <deal.II/base/parameter_handler.h>
-
 // ExaDG
+#include <exadg/utilities/parameter_handler.h>
 #include "print_solver_results.h"
```

Comment:
not related to this PR, but should for consistency read `<..../print_solver_results.h>`

Reply:
I filed a new PR for this https://github.com/exadg/exadg/pull/466